sofdev6-02-2024/LosAderezos
# Feat: No company page

### Test steps:
User with a company 
1. Log in to the app with a register user that is part of a company/subsidiary
2. Verify the view matches the mockups:
![image](https://github.com/user-attachments/assets/cf46ea3c-be15-4f0b-b401-ed7ac2dfd4e3)


User with no company
1. Log in to the app with a new user
2. Verify the view matches the mockups:
![image](https://github.com/user-attachments/assets/fe5d3ad0-4a97-4b68-b38e-a5004edae0ac)
